src/components/gui/gui.jsx

```diff
@@ -115,8 +115,8 @@ const GUIComponent = props => {
                 dir={isRtl ? 'rtl' : 'ltr'}
                 {...componentProps}
             >
-                {previewInfoVisible ? (
-                    <PreviewModal hideIntro={hideIntro} />
+                {(previewInfoVisible && !hideIntro) ? (
```

Comment:
You're not passing hideIntro into the component any more, so this should just be `previewInfoVisible`